tests/periph/freqm/main.c

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (C) 2023 ML!PA Consulting GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       Application to test functionality of the frequency meter
+ *              peripheral
+ *
+ * @author      Urs Gompper <urs.gompper@ml-pa.com>
+ * @}
+ */
+
+#include <stddef.h>
+#include <stdbool.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <stdint.h>
+
+#include "periph/freqm.h"
+
+volatile bool freqm_buffer_ready = false;
+uint32_t freqm_freq_hz = 0;
+bool freqm_overflow = false;
```

Comment:
```suggestion
static uint32_t freqm_freq_hz;
static bool freqm_overflow;
```

Reply:
You might as well make use of the context pointer instead of using global variables